engine/tests/phpunit/Elgg/TestCase.php

```diff
@@ -4,10 +4,16 @@
 
 use Elgg\Di\ServiceProvider;
 use Elgg\Http\Request;
+use PHPUnit_Framework_TestCase;
 use stdClass;
+use Symfony\Component\HttpFoundation\Response;
 use Zend\Mail\Transport\InMemory as InMemoryTransport;
 
-abstract class TestCase extends \PHPUnit_Framework_TestCase {
+/**
+ * @runTestsInSeparateProcesses
```

Comment:
You would think that this would slow down tests, but it made them faster. I think because our services are quite massive preserving global states in fact makes things worse.